packages/perps-exes/src/bin/perps-companion/types.rs

```diff
@@ -65,6 +67,7 @@ impl From<ChainId> for i32 {
             ChainId::Pion1 => 13,
             ChainId::RujiraTestnet => 14,
             ChainId::RujiraMainnet => 15,
+            ChainId::RujiraDevnet => 16,
```

Comment:
I don't think we'll be needing this, I doubt we'll be hosting a frontend for devnet. Can you ask on Discord for an ETA of enshrined oracles being available on stagenet?